ui/admin/tests/unit/utils/route-info-test.js

```diff
@@ -0,0 +1,9 @@
+import { paramValueFinder } from 'admin/utils/route-info';
+import { module, test } from 'qunit';
+
+module('Unit | Utility | route-info', function () {
+  test('it works', function (assert) {
```

Comment:
Is it too difficult to test this function? This would be something that would be great to have some unit tests for especially as it's recursive

Reply:
oop, no its not that hard to test, just completely forgot to add some tests